I18N.md

```diff
@@ -0,0 +1,55 @@
+## i18n
+
+See [i18n](link) for information on our internationalization tools and guidelines
+
+We use [react-i18next](https://react.i18next.com/) to translate messages in assisted-ui-lib. The
+i18n namespace must be declared in TRANSLATION_NAMESPACE property to avoid naming conflicts.
```

Comment:
We need to verify https://github.com/openshift-assisted/assisted-ui-lib/pull/1399/files#r883540666 , maybe we can provide some meaningful default for TRANSLATION_NAMESPACE  ?